internal/cmd/base/internal/docker/supported.go

```diff
@@ -41,20 +41,32 @@ func startDbInDockerSupported(dialect string, opt ...Option) (cleanup func() err
 		}
 	}
 
+	runOpts := &dockertest.RunOptions{
+		Tag: tag,
+		Env: []string{"POSTGRES_PASSWORD=password", "POSTGRES_DB=boundary"},
+		Cmd: []string{
+			// JIT seems to cause noticeable overhead without providing noticeable benefit.
```

Comment:
```suggestion
			// JIT seems to cause noticeable overhead without providing noticeable benefit, at least prior to Postgres 14.
```
Just a note in case we want to try it out again as we move to later versions, where possibly it actually becomes an improvement.

Reply:
I think JIT would be the same for postgres 14, it just so happens that starting with 14 a different plan is chosen that has a lower cost and so it does not trigger JIT. I can try it out by lowering the threshold so JIT is triggers with the plan for postgres 14 and see if it still produces the same overhead.

Reply:
Yea, just confirmed that I get the same slowness by lowering `jit_above_cost` so that JIT runs for the plan produced by postgres 14.